examples/federation/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
This does not need to be a Maven project so you can remove all of the Maven wrapper files, the `.mvn` directory, `mvnw` and `mvnw.cmd`